docs/elements/examples/primitives.mdx

```diff
@@ -0,0 +1,114 @@
+                <path d="M27 3.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM25 8.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM23 13.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM21.5 18.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM20.5 23.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM22.5 28.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM25 33.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2ZM27 38.5a1 1 0 1 0 0-2v2Zm0-2h-46v2h46v-2Z" />
+              </g>
+            </svg>
+            <h1 className="mt-4 text-xl font-medium tracking-tight text-zinc-950">Please check your email</h1>
+            <p className="text-sm text-zinc-500">
+              We’ve sent a code to <span className="font-medium text-zinc-950">hi@company.com</span>
+            </p>
+          </header>
+          <Clerk.GlobalError className="block text-sm text-red-400" />
+          <div className="space-y-4">
+            <Clerk.Field name="code" className="space-y-2">
+              <Clerk.Label className="sr-only">Email code</Clerk.Label>
+              <Clerk.Input
+                type="otp"
+                required
+                className="flex justify-center gap-x-1"
+                render={({ value, status }) => (
+                  <div
+                    className={`relative grid h-9 w-8 place-content-center rounded-md bg-white text-zinc-950 ring-1 ring-inset ring-zinc-300 ${status === 'selected' ? 'bg-sky-400/10 shadow-[0_0_8px_2px_theme(colors.sky.400/30%)] ring-sky-400' : ''}`}
+                  >
+                    <AnimatePresence>
+                      {value && (
+                        <motion.span
+                          initial={{ opacity: 0, scale: 0.75 }}
+                          animate={{ opacity: 1, scale: 1 }}
+                          exit={{ opacity: 0, scale: 0.75 }}
+                          className="absolute inset-0 flex items-center justify-center text-zinc-950"
+                        >
+                          {value}
+                        </motion.span>
+                      )}
+                      {value}
+                    </AnimatePresence>
+                    {status === 'cursor' && (
+                      <motion.div
+                        layoutId="otp-input-focus"
+                        transition={{ ease: [0.2, 0.4, 0, 1], duration: 0.2 }}
+                        className="absolute inset-0 z-10 rounded-[inherit] border border-sky-400 bg-sky-400/10 shadow-[0_0_8px_2px_theme(colors.sky.400/30%)]"
+                      />
+                    )}
+                  </div>
+                )}
+              />
+              <Clerk.FieldError className="block text-sm text-red-400" />
+            </Clerk.Field>
```

Comment:
Can we only show this in the code block? Everything else is just boilerplate I'd say

Reply:
Yep, exactly

Reply:
👍🏼 https://github.com/clerk/clerk-docs/pull/979/commits/ba409bc7195bb83e93c1fc64861e491c9f4f8f2b